docs/src/Nlay_tutorial.md

```diff
@@ -0,0 +1,151 @@
+
+```julia
+julia> using LightPropagation
+julia> fluence_DA_Nlay_cylinder_CW(1.0, (0.20, 0.15), (15.0, 20.0); n_ext=1.0, n_med=(1.0, 1.0), l=(0.4, 0.8), a=20.0, z=0)
+0.007359921451026246
+```
+
+NOTE: make sure you put all arguments in decimal point form, i.e. μsp= 15.0 not 15, or there will be an error.
+
+We can see that our output is a singular fluence value at an ρ of 1.0 cm. If we wanted to generate a series of data point in order to plot fluence against increasing ρ, for example from 0 to 1.0 cm, we could simply create a range for the ρ variable as follows:
+
+```julia
+julia> ρ = 0.1:0.1:1.0
+julia> cw_2lay = fluence_DA_Nlay_cylinder_CW(ρ, (0.20, 0.15), (15.0, 20.0); n_ext=1.0, n_med=(1.0, 1.0), l=(0.4, 0.8), a=20.0, z=0)
+```
+And then subsequently we can plot the data like so:
+
+```julia
+julia> using Plots	
+julia> plot(ρ, log10.(cw_2lay), xlabel="Source Detector Separation (cm)", ylabel="fluence (1/cm^2)", title="Steady-State Fluence",label="2-layer solution")
+```
+
+![2lay_plot](./assets/2lay_plot.png)
+
+### Time-domain example
+We may also want to simulate light propagation through layered media in the time-domain. In order to do this we can utilize the function `fluence_DA_Nlay_cylinder_TD` which has slightly different arguments than those for the steady-state example:
+
+```julia
+fluence_DA_Nlay_cylinder_TD(t, ρ, μa, μsp; n_ext=1.0, n_med=(1.0, 1.0), l=(1.0, 5.0), a=10.0, z=0.0, MaxIter=10000, atol=eps(Float64), N, ILT)
+```
+
+In this example we will focus on the required inputs, `t, ρ, μa, and μsp`, as well as the key word arguments `n_ext, n_med, l, a, z`, and `N`. For more information on what each of these arguments are, see the docs by typing `? fluence_DA_Nlay_cylinder_TD` in your julia window.
+
+Below are the values you would put into this function to model the 2-layered media  at a single time value (i.e. 1 ns) and ρ = 1.0 cm. Notice that μa, μsp, n_med, and l all must have the same number of arguments as you want layers in the model:
+
+```julia
+julia> fluence_DA_Nlay_cylinder_TD(1.0, 1.0, (0.20, 0.15), (15.0, 20.0); n_ext=1.0, n_med=(1.0, 1.0), l=(0.4, 0.8), a=20.0, z=0)
+2.2362727487107553e-5
+```
+
+To generate the fluence at multiple time values (i.e. 0.01 to 5.0 ns), enabling a simulation of fluence vs time, you can perform the following example:
+
+```julia
+julia> t = 0.01:0.1:3.0
+julia> td_2lay = fluence_DA_Nlay_cylinder_TD(t, 1.0, (0.20, 0.15), (15.0, 20.0); n_ext=1.0, n_med=(1.0, 1.0), l=(0.4, 0.8), a=20.0, z=0, N=72)
+```
+
+![2lay_plot_td](./assets/2lay_plot_td.png)
+
+NOTE: Here, I entered in N=72 manually instead of leaving out the keyword argument since an oscillating solution was seen with the default value. An example below shows what the plot would look like with the defaulted N value:
```

Comment:
I would probably detail what N is explicitly in this example then. And mention that a larger value of N will be needed because t[end]/t[1] is large.